src/protect/mutation_calling/fusion.py

```diff
@@ -13,20 +13,407 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from __future__ import print_function
+from math import ceil
```

Comment:
There's multiple 100+ char lines in this file. not critical  but since you're making other changes, do this too.